content/applications/sales/sales/invoicing/time_materials.rst

```diff
@@ -1,136 +1,370 @@
-suppliers.
+This is different from a fixed-price contract, when a customer agrees to pay a specified total for
+the fulfillment of the contract---no matter what needs to be paid to the employees, sub-contractors,
+vendors, suppliers, and so on.
 
-For this documentation I will use the example of a consultant, you will
-need to invoice their time, their various expenses (transport,
-lodging, ...) and purchases.
+The Odoo *Sales* app can invoice for time and various other expenses (e.g. transport, lodging), as
+well as purchases needed to fulfill an order.
 
-Invoice time configuration
-==========================
+App and settings configuration
+==============================
 
-To keep track of progress in the project, you will need the *Project*
-app. Go to :menuselection:`Apps --> Project` to install it.
+First, in order to accurately keep track of the progress of a project, the Odoo *Project* and
+*Accounting* apps **must** be installed.
 
-In *Project* you will use timesheets, to do so go to
-:menuselection:`Project --> Configuration --> Settings` and activate the
-*Timesheets* feature.
+To install the *Project* app, navigate to :menuselection:`Odoo main dashboard --> Apps`. Then,
+on the :guilabel:`Apps` page, locate the :guilabel:`Project` app block, and click
+:guilabel:`Activate`. The page automatically refreshes and returns to the main Odoo dashboard, where
+the *Project* app is now available to access.
 
-.. image:: time_materials/time_materials01.png
+Repeat the same process to install the *Accounting* application.
+
+After installation, click the :guilabel:`Accounting` app icon from the main Odoo dashboard, and
+navigate to :menuselection:`Configuration --> Settings`. On the :guilabel:`Settings` page, scroll
+down to the :guilabel:`Analytics` section, and ensure the box next to :guilabel:`Analytic
+Accounting` is checked.
+
+.. image:: time_materials/analytic-accounting-setting.png
    :align: center
+   :alt: How it looks to activate the Analytic Accounting setting in Odoo Accounting Setting page.
 
-Invoice your time spent
-=======================
+Then, click :guilabel:`Save` to save all changes.
 
-From a product page set as a service, you will find two options under
-the invoicing tab, select both *Timesheets on tasks* and *Create a
-task in a new project*.
+Then, navigate to :menuselection:`Odoo main dashboard --> Project app --> Configuration -->
+Settings`. On the :guilabel:`Settings` page, in the :guilabel:`Time Management` section, ensure the
+box beside the :guilabel:`Timesheets` feature is checked.
 
-.. image:: time_materials/time_materials02.png
+Then, click :guilabel:`Save` to save all changes.
+
+.. image:: time_materials/timesheets-feature.png
    :align: center
+   :alt: What the Timesheets feature looks like on the Odoo Project settings page.
 
-You could also add the task to an existing project.
+.. _sales/invoicing/time_materials/configured-service-product:
```

Comment:
No need to add the doc name _/time_materials/_ in the custom anchor, the app name and parent folder are enough 🙂
```suggestion
.. _sales/invoicing/configured-service-product:
```